src/UI/SlotArea.cpp

```diff
@@ -1114,6 +1116,7 @@ void cSlotAreaAnvil::UpdateResult(cPlayer & a_Player)
 				Output.Empty();
 				SetSlot(2, a_Player, Output);
 				m_ParentWindow.SetProperty(0, 0, a_Player);
+				m_ParentWindow.SendSlot(a_Player, this, 2);
```

Comment:
Still this feels wrong. If you replace all the `SendSlot` calls with `SetSlot`, does it work?